src/potions.ts

```diff
@@ -493,6 +560,20 @@ export function doublingPotions(embezzlers: number): Potion[] {
     .map((pair) => pair.potion);
 }
 
+export function usePawWishes(singleUseValuation: (potion: Potion) => number): void {
+  while (get("_pawWishes", 0) < 5) {
+    // Sort the paw potions by the profits of a single wish, then use the best one
+    if (
+      !pawPotions
+        .filter((potion) => numericModifier(potion.effect(), "Meat Drop") >= 100)
+        .sort((a, b) => singleUseValuation(b) - singleUseValuation(a))
+        .some((potion) => potion.use(1))
```

Comment:
Don't put effectful code in this `if` statement

Reply:
Or, rather, not this far buried in the `if` statement--obviously you need to capture and evaluate the value somehow